NFC OOB pairing fixes (#9926)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    * Added energy consumption information to documentation.
    * Added a documentation section about distance offset calibration.

> * :ref:`peripheral_nfc_pairing` and :ref:`central_nfc_pairing` samples:

Reviewer: Isn't the fix in the AC Parser library? Maybe it should rather be mentioned in the NFC section?

Author: Maybe we can mention it in both places?

Reviewer: Both are OK for me.

Author: Ok I added notes in both places

    * Added energy consumption information to documentation.
    * Added a documentation section about distance offset calibration.

* :ref:`peripheral_nfc_pairing` and :ref:`central_nfc_pairing` samples:

>   * Fixed an OOB pairing between these samples.

Reviewer 2: ```suggestion
  * Fixed OOB pairing between these samples.
```

NFC samples
-----------

* Fixed:

>   * An issue where NFC samples using a NFC Reader feature, returned false error code with value 1 during NFC T4T operation.

Reviewer 2: ```suggestion
  * An issue where NFC samples using the NFC Reader feature, returned false error code with value ``1`` during NFC T4T operation.
```

>   * An issue where NFC samples using the NFC Reader feature, returned false error code with value ``1`` during NFC T4T operation.

Reviewer 3: ```suggestion
  * An issue where NFC samples that use the NFC Reader feature returned false error code with value ``1`` during the NFC T4T operation.
```
